src/main/java/se/sundsvall/supportmanagement/service/MetadataService.java

```diff
@@ -38,33 +46,84 @@ public class MetadataService {
 	@Autowired
 	private ValidationRepository validationRepository;
 
-	@Cacheable(value = "metadataCache", key = "{#root.methodName, #namespace, #municipalityId}")
+	// =================================================================
+	// Common operations
+	// =================================================================
+
+	@Cacheable(value = CACHE_NAME, key = "{#root.methodName, #namespace, #municipalityId}")
 	public MetadataResponse findAll(String namespace, String municipalityId) {
 		return MetadataResponse.create()
 			.withCategories(findCategories(namespace, municipalityId))
 			.withStatuses(findStatuses(namespace, municipalityId))
 			.withExternalIdTypes(findExternalIdTypes(namespace, municipalityId));
 	}
 
-	@Cacheable(value = "metadataCache", key = "{#root.methodName, #namespace, #municipalityId}")
+	@Cacheable(value = CACHE_NAME, key = "{#root.methodName, #namespace, #municipalityId, #type}")
+	public boolean isValidated(String namespace, String municipalityId, EntityType type) {
+		return validationRepository.findByNamespaceAndMunicipalityIdAndType(namespace, municipalityId, type)
+			.map(ValidationEntity::isValidated)
+			.orElse(false);
+	}
+
+	// =================================================================
+	// ExternalIdType operations
+	// =================================================================
+
+	@Cacheable(value = CACHE_NAME, key = "{#root.methodName, #namespace, #municipalityId}")
 	public List<ExternalIdType> findExternalIdTypes(String namespace, String municipalityId) {
 		return externalIdTypeRepository.findAllByNamespaceAndMunicipalityId(namespace, municipalityId)
 			.stream()
 			.map(MetadataMapper::toExternalIdType)
 			.filter(Objects::nonNull)
+			.sorted((o1, o2) -> ObjectUtils.compare(o1.getName(), o2.getName()))
```

Comment:
Another solution, (which is a little more readable i.m.h.o.)
`.sorted(comparing(ExternalIdType::getName))`

Reply:
Agree, will change